src/docs/sphinx/ARM.rst

```diff
@@ -4,6 +4,70 @@
 .. # SPDX-License-Identifier: MIT
 
 
-ARM Overview (under development)
+Arm Overview
 ================================
 
+This implementation of the Arm port of Variorum supports the Arm Juno r2 SoC.
+The Arm Juno r2 platform is a big.LITTLE cluster with Cortex-A72 (big) and
+Cortex-A53 (LITTLE) clusters (also called processors), respectively.
+It also has an Arm Mali GP-GPU. We have tested the Arm functionality of
+Variorum on Linaro supported Linux.
+
+References:
+
+* `Arm Juno r2 technical reference <https://developer.arm.com/documentation/100114/0200/>`_
+
+Monitoring and Control Through Sysfs Interface
+-----------------------------------------------
+
+The built-in monitoring interface on the Arm Juno r2 board is implemented
+by the on-board FPGA. Since this interface is not universally available
+on most Arm implementations, we leverage the standard Linux sysfs interface
+for monitoring and control.
+Following lists the specific metrics that are monitored on Arm Juno r2:
+
+* Power telemetry:
+
+The sysfs interface provides a file for each of the following
+Advanced Peripheral Bus (APB) energy meter registers:
+
+    * SYS_POW_SYS : /sys/class/hwmon/hwmon0/power1_input
+    * SYS_POW_A72 : /sys/class/hwmon/hwmon0/power2_input
+    * SYS_POW_A53 : /sys/class/hwmon/hwmon0/power3_input
+    * SYS_POW_GPU : /sys/class/hwmon/hwmon0/power4_input
+
+Instantaneous power is recorded in bits 0-23 and are reported in microwatts by the
+sysfs interface. Variorum reports power in milliwatts. All registers are updated every
```

Comment:
Is there a reason Variorum is converting the native microwatts to milliwatts? Should variorum just report the power in milliwatts?

Reply:
The verbose reporting of power in microwatts looked too long so it's reported in milliwatts. Should it report raw values?

Reply:
Todo: Add notes about why we're doing the conversion in docs and code.